conf/route_control.py

```diff
@@ -337,7 +349,7 @@ def parse_new_route(msg):
 
     # if mac is 0, send ARP request
     if gateway_mac == 0:
-        print('Adding entry {} in arp probe table'.format(item.iface))
+        print('Adding entry {} in arp probe table. Neighbor - {}'.format(item.iface,item.neighbor_ip))
```

Comment:
What about `Neighbor is` or `Neighbor:` instead of `Neighbor -`?
```suggestion
        print('Adding entry {} in arp probe table. Neighbor: {}'.format(item.iface,item.neighbor_ip))
```